fusor-ember-cli/app/routes/review/installation.js

```diff
@@ -121,8 +122,47 @@ export default Ember.Route.extend(NeedsExistingManifestHelpers, {
       },
       data: {}
     }).then(response => {
-      controller.set('validationErrors', response.validation.errors);
-      controller.set('validationWarnings', response.validation.warnings);
+       if (Ember.isPresent(response.validation)) {
+         controller.set('validationErrors', response.validation.errors);
+         controller.set('validationWarnings', response.validation.warnings);
+       }
+       console.log('validate() was successful');
+       return Ember.RSVP.Promise.resolve('validate() was successful');
```

Comment:
@cfchase, I explicitly returned a promise on success here. Is this what you were referring to?